build_projects/dotnet-host-build/DebTargets.cs

```diff
@@ -267,5 +267,10 @@ private static bool DebuildNotPresent()
         {
             return Cmd("/usr/bin/env", "debuild", "-h").Execute().ExitCode != 0;
         }
+
+        private static bool ShouldSkipBuildDebPackages()
+        {
+            return Environment.GetEnvironmentVariable("DOTNET_BUILD_SKIP_DEB_PACKAGING") == "1";
```

Comment:
I assumed the way this would all be handled would change with the move to build tools.  I can add a switch that sets this environment variable if you'd prefer to build.sh